xla/service/gpu/fusions/dynamic_slice_fusion_test.cc

```diff
@@ -3190,6 +3190,61 @@ TEST_F(DynamicSliceFusionTest, ReduceScatterDynamicSlice) {
                                                 false, true, error));
 }
 
+TEST_F(DynamicSliceFusionTest, OffsetComputation) {
```

Comment:
Can you make the test name explain clearly what behaviour is tested?

This seems to be about some `OffsetComputation` being supported. What exactly is that? `OffsetsThatCanBeEvaluatedSuccessfullyAreCorrectlyEmbeddedIntoThunks`?